src/SignIn.js

```diff
@@ -10,23 +12,36 @@ class SignIn extends Component{
     super();
     this.state = {
       username: '',
-      password: ''
+      password: '',
+      loginAttempt: false,
+      error: false,
     };
     this.onChange = this.onChange.bind(this);
```

Comment:
I think it'll be a better naming convention if you do onChangeSomething

Reply:
I will update this